tox_binary_io/src/lib.rs

```diff
@@ -140,13 +138,13 @@ macro_rules! unpack {
     ($variable:expr, $variant:path [ $($inner:ident),* ]) => (
         match $variable {
             $variant( $($inner),* ) => ( $($inner),* ),
-            other => panic!("Expected {} but got {:?}", stringify!($variant), other),
```

Comment:
Why did you remove `other` output?

Reply:
Because it doesn't implement `Debug` anymore (because secret/precomputed keys from rust-crypto don't implement it).